### DISCUSSION TITLE
Transitioning from MinIO to AWS S3 for TerraKube in docker

### DISCUSSION THREAD

**Discussion by abhinavsquadstack** [terrakube-io/terrakube#490] [Q&A] [answered]
Hello Team,
I've successfully deployed TerraKube using MinIO, and it's currently working as expected. Now, I'd like to migrate the state file, registry, and plan to an AWS S3 bucket instead of MinIO. Could you please provide guidance on the correct process for this migration? I've reviewed the documentation, but it doesn't mention the steps specifically for a Docker implementation.

Thanks

**Comment by alfespa17** [comment, marked as answer]
Hello @abhinavsquadstack 

Maybe you could use MinIO Console (http://localhost:9001), so you can download the files

![image](https://github.com/AzBuilder/terrakube/assets/4461895/9e4ee440-9ba2-4bbc-afdd-1382d40ccdc3)

Once you have the zip file for each folder you could upload that to your S3

I guess that could work for you

**Reply by abhinavsquadstack** [reply]
In env files there is a variable `AwsEndpoint`, for now its value is minio url, but for aws S3 bucket do I need to put the hosted s3's url?

**Reply by alfespa17** [reply]
You can leave that value empty, you only need to use it with MinIO
